### PR TITLE
Comma separated list for programs on success screen

### PR DESCRIPTION
**Pull request by andrewedstrom**
https://www.pivotaltracker.com/story/show/178115548

### CODE REVIEW

File: src/main/java/org/codeforamerica/shiba/Utils.java [hunk 1]
```diff
@@ -10,6 +10,10 @@
 
 @Slf4j
 public class Utils {
+    private Utils() {
+        throw new AssertionError("Cannot instantiate utility class");
+    }
```
Comment (bencalegari) [inline src/main/java/org/codeforamerica/shiba/Utils.java:15]: This is to encourage static references and prevent instantiation right?

**Review by bencalegari — APPROVED** [review]

File: src/main/java/org/codeforamerica/shiba/Utils.java (hunk 1 above)
Comment (andrewedstrom) [reply src/main/java/org/codeforamerica/shiba/Utils.java:15]: Yeah exactly. I realized our utility classes were instantiable and decided to add this guard just to make sure we don't enable future devs to do something weird.

File: src/main/java/org/codeforamerica/shiba/pages/SuccessMessageService.java [hunk 2]
```diff
@@ -19,80 +21,78 @@ public SuccessMessageService(MessageSource messageSource) {
...
 
-        return nextStepLetterProgramNames;
+        return nextStepLetterPrograms;
     }
 
-    @NotNull
-    private String getMessage(String messageKey, Locale locale) {
-        return messageSource.getMessage(messageKey, null, locale);
+    private boolean hasProgram(String program, List<String> programs) {
+        return programs.stream().anyMatch(p -> p.equals(program));
```
Comment (sree-cfa) [inline src/main/java/org/codeforamerica/shiba/pages/SuccessMessageService.java:96]: Could this just be a `programs.contains(program)`?

Comment (andrewedstrom) [reply src/main/java/org/codeforamerica/shiba/pages/SuccessMessageService.java:96]: YES great point, thank you! I'll change that

**Pull request merged**